Corona/Localization/fr.lproj/Localizable.strings

```diff
@@ -0,0 +1,30 @@
+/*
+  Localizable.strings
+  Corona
+
+  Created by Mohammad on 3/22/20.
+  Translation by Clément Knoderer - github.com/Risengan
+  Copyright © 2020 Samabox. All rights reserved.
+*/
+
+"case.confirmed" = "Confirmés";
+"case.active" = "Hospitalisés";
+"case.recovered" = "Réanimés";
+"case.deaths" = "Morts";
+"region.world" = "Monde";
+"chart.topCountries" = "Pays les plus touchés";
+"chart.logarithmic" = "Échelle logarithmique";
+"data.updating" = "Mise à jour...";
+"data.updateDate" = "Dernière mise à jour :";
+"data.errorTitle" = "Impossible de mettre à jour les données";
+"data.errorMessage" = "Veuillez vous assurer que vous êtes connecté à Internet.";
+"app.newVersionTitle" = "Nouvelle version disponible";
+"app.newVersionMessage" = "Veuillez mettre à jour à partir de %@";
+"message.cancel" = "Annuler";
+"message.open" = "Ouvrir";
+"message.ok" = "OK";
+"message.done" = "OK";
```

Comment:
I guess "Done" should not be the same as "OK".

This is the context:

![image](https://user-images.githubusercontent.com/121827/77454240-50591d80-6e09-11ea-994e-0057c6b4bbfe.png)

To see it: Tap the (⠸) button, then "Share"

Reply:
It’s the best choice in French 😉

Reply:
Yeah, it looks like Apple is also using OK for both OK & Done. Thanks!